src/js/stores/PluginStore.js

```diff
@@ -1,78 +1,121 @@
 import {EventEmitter} from "events";
 
 import AppDispatcher from "../AppDispatcher";
-import PluginDispatcher from "../plugin/external/PluginDispatcher";
+import Util from "../helpers/Util";
 import DialogActions from "../actions/DialogActions";
 import DialogSeverity from "../constants/DialogSeverity";
 import PluginActions from "../actions/PluginActions";
 import PluginEvents from "../events/PluginEvents";
+import PluginModules from "../constants/PluginModules";
+import States from "../constants/States";
+import pluginScheme from "../stores/schemes/pluginScheme";
 
-import Util from "../helpers/Util";
+var plugins = [];
+
+function addPlugin(data) {
+  if (data.id == null && !Util.isObject(data.info)
+    && getPluginById(data.id) != null) {
```

Comment:
Actually, how do you break logical-operators? Cannot find anything in our coding guidelines.
I though the `&&` should dangle at the line above. But call me wrong.

At least this line needs +2 ident.